### PR TITLE
Recover account

### PR DESCRIPTION
**Pull request by vgrichina**
Follow-up to https://github.com/nearprotocol/near-wallet/pull/29

### CODE REVIEW

File: src/components/accounts/RecoverAccount.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+   handleChange = (e, { name, value }) => {
+      this.setState(() => ({
+         [name]: value,
+         isLegit: this.isLegitField(name, value)
+      }))
+   }
+
+   isLegitField(name, value) {
+      // TODO: Use some validation framework?
```
Comment (potatodepaulo) [inline src/components/accounts/RecoverAccount.js]: https://www.npmjs.com/package/react-validation

Comment (vgrichina) [reply src/components/accounts/RecoverAccount.js]: looks pretty cool, thanks!

Comment (janedegtiareva) [reply src/components/accounts/RecoverAccount.js]: let's not use "isLegitField". It's not common language for this. https://www.merriam-webster.com/thesaurus/leg%20(it)
the word to use here is "valid"

Comment (vgrichina) [reply src/components/accounts/RecoverAccount.js]: you are linking verb definition, which has no relation to "legit" adjective:

Screen Shot 2019-05-16 at 12 43 27 PM

**Pull request merged**